frontend/integration-tests/views/project-status.view.ts

```diff
@@ -0,0 +1,5 @@
+import { $, $$ } from 'protractor';
+
+export const softwareInfo = $('.col-lg-4  .group .group__title .h3');
+export const kubernetes = $('.col-lg-4  .group .group__body .row .col-xs-6');
+export const kubernetesVersion = $('.col-lg-4  .group .group__body .row .text-right div');
```

Comment:
These selectors assume too much about the exact structure of the DOM. This will be very fragile and difficult to maintain.

Reply:
@spadgett Thanks for remind! Updated it.